### PR TITLE
add test for empty bundles

### PR DESCRIPTION
**Pull request by MDunitz**
addresses https://github.com/HumanCellAtlas/dcplib/issues/32

### CODE REVIEW

File: dcplib/etl/__init__.py [hunk 1]
```diff
@@ -93,8 +93,10 @@ def extract(self, query=None, max_workers=512, max_dispatchers=1,
 
                 if loader is not None:
                     for future in concurrent.futures.as_completed(futures):
-                        loader(bundle=future.result())
-                        loaded_bundle_count += 1
+                        bundle = future.result()
+                        if bundle:
```
Comment (kislyuk) [inline dcplib/etl/__init__.py]: It's safer here to explicitly compare the bundle to None.
